Obsidian/Client.cs

```diff
@@ -113,7 +114,7 @@ public sealed class Client : IDisposable
     /// <summary>
     /// The socket associated with the <see cref="networkStream"/>.
     /// </summary>
-    private readonly Socket socket;
+    private readonly ConnectionContext socket;
```

Comment:
Let's go ahead and rename socket to connectionContext throughout

Reply:
Done!